src/components/job-common.js

```diff
@@ -172,22 +173,52 @@ export const getTaskCost = ({ vmCostUsd, taskStartTime, taskEndTime }) => {
 };
 
 export const renderTaskCostElement = (cost) => {
-  if (cost === 0.0) {
+  if (cost.toFixed(2) === '0.00') {
     return '< $0.01';
   }
   return `$${cost.toFixed(2)}`;
 };
 
-export const calculateTotalCost = (callObjects) => {
+export const calculateTotalCost = async (callObjects, loadForSubworkflows) => {
   let total = 0;
-  Object.values(callObjects).forEach((call) => {
-    if (!call[0].taskStartTime) {
-      total += 0;
-    } else if (call[0].taskEndTime) {
-      total += getTaskCost({ vmCostUsd: call[0].vmCostUsd, taskStartTime: call[0].taskStartTime, taskEndTime: call[0].taskEndTime });
-    } else {
-      total += getTaskCost({ vmCostUsd: call[0].vmCostUsd, taskStartTime: call[0].taskStartTime });
+  let callObs = callObjects;
+
+  if (!_.isNil || callObjects?.subWorkflowId) {
```

Comment:
maybe there's something going on here that I don't understand, but is this call to `_.isNil` actually doing anything? it looks like it's checking for the existence of the function in lodash, not actually being invoked

Reply:
No you're right. I think the second half of this got last in a refactor! Will be fixed in the next commit!